api/proto/node/node.proto

```diff
@@ -27,12 +27,32 @@ service Dispersal {
 }
 
 service Retrieval {
-	// RetrieveChunks retrieves the chunks for a blob custodied at the Node.
-	rpc RetrieveChunks(RetrieveChunksRequest) returns (RetrieveChunksReply) {}
+	// GetChunks retrieves all of the chunks for a blob held by the node.
+	rpc GetChunks(GetChunksRequest) returns (GetChunksReply) {}
+	// GetChunk retrieves a specific chunk for a blob custodied at the Node.
+	rpc GetChunk(GetChunkRequest) returns (GetChunkReply) {}
+	// StreamChunks requests a stream of chunks with a specific index.
+	rpc StreamChunks(stream StreamChunksRequest) returns (stream GetChunkReply) {}
 	// GetBlobHeader is similar to RetrieveChunks, this just returns the header of the blob.
 	rpc GetBlobHeader(GetBlobHeaderRequest) returns (GetBlobHeaderReply) {}
-	// Retrieve node info metadata
-	rpc NodeInfo(NodeInfoRequest) returns (NodeInfoReply) {}
+	// StreamBlobHeaders streams the blob headers to the client.
+	rpc StreamBlobHeaders(StreamBlobHeadersRequest) returns (stream GetBlobHeaderReply) {}
```

Comment:
I think we can just include the blob headers in the `StreamChunks` stream.

Reply:
removed

Reply:
This is added back in per our most recent discussions.